x-pack/plugins/security_solution_serverless/public/get_started/get_started.tsx

```diff
@@ -73,17 +78,32 @@ export const GetStartedComponent: React.FC<GetStartedProps> = ({ productTypes })
               padding-left: ${euiTheme.size.xs};
             `}
           >
-            <span>{GET_STARTED_PAGE_TITLE}</span>
+            <>
+              <span>{GET_STARTED_PAGE_TITLE}</span>
+              <Spacer />
```

Comment:
If using EuiSpacer causes the error to appear, here we can add `[padding/margin]-bottom` to the EuiTitle.